models/login_source.go

```diff
@@ -403,6 +403,20 @@ func LoginViaLDAP(user *User, login, password string, source *LoginSource, autoR
 
 	var isAttributeSSHPublicKeySet = len(strings.TrimSpace(source.LDAP().AttributeSSHPublicKey)) > 0
 
+	// Update User if exist
+	isExist, err := IsUserExist(0, sr.Username)
+	if err != nil {
+		return nil, err
+	} else if isExist {
+		user.FullName = composeFullName(sr.Name, sr.Surname, sr.Username)
+		user.Email = sr.Mail
+		// Change existing admin flag only if AdminFilter option is set
+		if len(source.LDAP().AdminFilter) > 0 {
+			user.IsAdmin = sr.IsAdmin
+		}
+	UpdateUserCols(user, "full_name", "email", "is_admin")
```

Comment:
```suggestion
		UpdateUserCols(user, "full_name", "email", "is_admin")
```
Let's see if this change fixes the fmt. Otherwise you'll have to pull the current version to your local server, run `make fmt` and push your results back.

Reply:
@DblK sory mi diff didnt take care of fmt "indent" @guillep2k thanks :D